src/NServiceBus.Unity/PropertyInjectionContainerExtension.cs

```diff
@@ -14,7 +15,13 @@ public PropertyInjectionContainerExtension(UnityObjectBuilder unityObjectBuilder
 
         protected override void Initialize()
         {
-            Context.Strategies.Add(new PropertyInjectionBuilderStrategy(unityObjectBuilder), UnityBuildStage.Initialization);
+            propertyInjectionStrategy = new PropertyInjectionBuilderStrategy(unityObjectBuilder);
+            Context.Strategies.Add(propertyInjectionStrategy, UnityBuildStage.Initialization);
+        }
+
+        public override void Remove()
+        {
+            propertyInjectionStrategy.Stop();
```

Comment:
Could we remove the strategy here instead of stopping it? (not sure, don't remember the Unity API that well)

Reply:
Not in this version of Unity. The next version (separate PR after this release) allows that (I think).